### PR TITLE
Adding an endpoint for case comment range queries

### PR DESCRIPTION
**Pull request by engineersamuel**
Changes include:
- New endpoint for case comment range queries for Ascension Chatter
- Webstorm re-formatting to make the js style more consistent (This was automatic)
- Fixed the test cases so they run and pass, use https://prod.foo.redhat.com/test/index.html, the accessproxy, and python -m SimpleHTTPServer 8080 in the udsjs dir
- One major change was to upgrade jsuri to 1.3.1, everything that relies on udsjs must also use this version now.
- version bump to 1.2.29

### CODE REVIEW

**Comment by engineersamuel** [comment]
@vrathee @gandhikeyur @jtrantin please review.

File: uds.js [hunk 1]
```diff
@@ -155,293 +155,295 @@
...
 
-    uds.getCaseGroupsForContact = function(contactSSO){
+    uds.getCaseGroupsForContact = function (contactSSO) {
         var url = udsHostName.clone().setPath('/case/casegroups/user/' + contactSSO);
-        return executeUdsAjaxCall(url,'GET');
+        return executeUdsAjaxCall(url, 'GET');
     };
 
-    uds.getRMECountForAccount = function(uql){
-        var url = udsHostName.clone().setPath('/case/history').addQueryParam('where', encodeURIComponent(uql));
```
Comment (vrathee) [inline uds.js:343]: @engineersamuel  Any specific reason for removing "encodeURIComponent" at multiple places?

Comment (engineersamuel) [inline uds.js:343]: @vrathee It is no longer required with jsuri 1.3.1.  I believe that the latest version of jsuri auto-escapes query params.  If we encode those query params then the result is double encoded, which breaks things.

Comment (vrathee) [inline uds.js:343]: Cool! Reviewed!

**Comment by jtrantin** [comment]
Reviewed.

**Pull request merged**